packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -71,7 +71,7 @@ export const WfoStartPage = () => {
             subscriptionsSummaryResult?.subscriptions.page.map(
                 (subscription) => ({
                     title: subscription.description,
-                    value: getFirstUuidPart(subscription.subscriptionId),
+                    value: formatDate(subscription?.startDate),
```

Comment:
The possibly undefined comment applies here too. The `subscription`, `workflow` and `task` cannot be undefined

Reply:
ok